### ISSUE TITLE
Give a bootstrap standard layout to JS Example

### ISSUE THREAD

**Issue by bensonby** [SkygearIO/skygear-SDK-JS#84]
http://sdkjsexample.skygeario.com/static/example/record.html

Give something looks "normal"

**Comment by railsstudent** [comment]
This page is really ugly.  Do you require styling for all html pages in example folder?

**Comment by chpapa** [comment]
@railsstudent yes. don't have to be fancy. ;) some short intro would be good too but not necessary. Can you help? :)

**Comment by railsstudent** [comment]
My name is Connie Leung and I started working on  front-end development last year. I use AngularJS, JavaScript/ES6 and Bootstrap/CSS at work. 
I also help out in HKOSCon and PyCon HK in my spare time
